db/schema.rb

```diff
@@ -26,106 +27,119 @@
     t.integer "inst_num", default: 1, null: false
     t.decimal "latitude", precision: 8, scale: 6
     t.decimal "longitude", precision: 9, scale: 6
-    t.integer "option_id"
-    t.integer "questioning_id"
+    t.integer "old_id", default: "nextval('answers_id_seq'::regclass)"
```

Comment:
This is interesting. What is the purpose of this?

Reply:
actually i need to fix that, ids retain their "create the next thing in the sequence" behavior even after being removed as primary keys and their name change, i forgot to reset the defaults for those fields, `schema.rb` doesn't actually support setting them this way so if you tried to load this schema and then do another migration, the next dump would have no default value for those columns, but as it stands, `old_id` will be non-null and keep counting up for new records post-migration for existing installations